bankAccount-system/index.js

```diff
@@ -0,0 +1,150 @@
+class BankAccount {
+    constructor(accountNumber, firstName, lastName, balance, dailyWithdrawalLimit) {
```

Comment:
Te user balance shouldn't be something you can manually set. as when you create a bank account in real life at a bank there is no option to specify an amount your account will have automatically, that will just be free money.